docs: add debian ci/ubuntu ppa links to README.md

kmk3 opened the pull request:
The Debian CI site is apparently where the official Debian packages are
built.

Relates to #6702 #6842.

reinerh commented:
Looks good. But for clarification, salsa is not for building the official packages that land in the repo, it's really just a CI for packaging.
This might also be interesting as a starting point when interested in the packaging: https://tracker.debian.org/pkg/firejail
It links to salsa, the build server that build the packages for the repo, the bug tracker etc...

kmk3 commented:
> Looks good. But for clarification, salsa is not for building the official
> packages that land in the repo, it's really just a CI for packaging.

I see; changed the text.

> This might also be interesting as a starting point when interested in the
> packaging: https://tracker.debian.org/pkg/firejail It links to salsa, the
> build server that build the packages for the repo, the bug tracker etc...

Nice, added this link as well.
